BrownianMotion/Gaussian/StochasticProcesses.lean

```diff
@@ -0,0 +1,62 @@
+import Mathlib
+
+open MeasureTheory
+
+variable {T Ω E : Type*} {mΩ : MeasurableSpace Ω} {P : Measure Ω}
+    {X Y : T → Ω → E}
+
+lemma modification_of_indistinduishable (h : ∀ᵐ ω ∂P, ∀ t, X t ω = Y t ω) :
+    ∀ t, ∀ᵐ ω ∂P, X t ω = Y t ω := by
+  intro t
+  filter_upwards [h] with ω hω using hω t
+
+variable [MeasurableSpace E]
+
+lemma finite_distributions_eq {n : ℕ} {t : Fin n → T} (h : ∀ t, ∀ᵐ ω ∂P, X t ω = Y t ω) :
+  Measure.map (fun ω m ↦ X (t m) ω) P = Measure.map (fun ω m ↦ Y (t m) ω) P := by
+  have h': ∀ᵐ (ω : Ω) ∂P, ∀ (m : Fin n), X (t m) ω = Y (t m) ω := by
+    rw [MeasureTheory.ae_all_iff]
+    intro i
+    exact h (t i)
+  refine Measure.map_congr ?_
+  filter_upwards [h']
+  simp_all only [Filter.eventually_all, implies_true, Set.mem_setOf_eq, Set.setOf_true,
+    Filter.univ_mem]
+
+lemma finite_distributions_eq_iff_same_law (hX : ∀ (t : T), Measurable (X t))
```

Comment:
AEMeasurable instead of Measurable should be enough, and we will need that generality.